music21/test/testStream.py

```diff
@@ -1418,11 +1418,9 @@ def testMeasureOffsetMap(self):
         # offset here is that of measure that originally contained this note
         self.assertEqual(sorted(list(mOffsetMap.keys())), [8.0])
 
-        # this should work but does not yet
-        # it seems that the flat score does not work as the flat part
-#         mOffsetMap = a.flat.measureOffsetMap('Note')
-#         self.assertEqual(sorted(mOffsetMap.keys()),
-#                [0.0, 4.0, 8.0, 12.0, 16.0, 20.0, 24.0, 28.0, 32.0]  )
+        mOffsetMap = a.flat.measureOffsetMap('Note')
+        self.assertEqual(sorted(mOffsetMap.keys()),
+                         [0.0, 4.0, 8.0, 12.0, 16.0, 20.0, 24.0, 34.0, 38.0])
```

Comment:
Are these numbers definitely right?  the failing test was for 20, 24, 28, 32 -- I can't remember how this works, but want to make sure it's right

Reply:
This chorale has a psalm tone, so these numbers looked correct when I opened the chorale.